www/apps/book/app/learn/fundamentals/admin/ui-routes/page.mdx

```diff
@@ -145,6 +145,32 @@ Some caveats for nested UI routes in the sidebar:
 - Nested routes in setting pages aren't shown in the sidebar to follow the admin's design conventions.
 - The `icon` configuration is ignored for the sidebar item of nested UI route to follow the admin's design conventions.
 
+
+### Route Under Existing Admin Route
```

Comment:
```suggestion
### Route Under Existing Admin Route

```

Reply:
@eugenepro2 there's also this change please (need to add a new line after the header)